app/views/organizations/edit.html.haml

```diff
@@ -0,0 +1,14 @@
+#main.ui.main.container
+  Edit Organization
+  = form_for @organization, html:{ class: "ui form" } do |f|
+    = label :name, "Name"
+    = f.text_field :name
+    = label :email, "E-mail"
+    = f.text_field :email
+    = label :location, "Location"
+    = f.text_field :location
+    = label :short_name, "Abbreviation"
```

Comment:
The abbreviation and short name are different fields, but they get mixed together here. Let's clarify this.